settings.gradle.kts

```diff
@@ -44,23 +44,24 @@ dependencyResolutionManagement {
 
             library("slf4j-api", "org.slf4j:slf4j-api:2.0.7")
 
-            library("jackson-dataformat-csv", "com.fasterxml.jackson.dataformat:jackson-dataformat-csv:2.14.2")
-            library("jackson-databind", "com.fasterxml.jackson.core:jackson-databind:2.14.2")
-            library("jackson-datatype-jsr310", "com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.14.2")
-            library("jackson-kotlin", "com.fasterxml.jackson.module:jackson-module-kotlin:2.14.2")
+            library("jackson-dataformat-csv", "com.fasterxml.jackson.dataformat:jackson-dataformat-csv:2.15.1")
+            library("jackson-databind", "com.fasterxml.jackson.core:jackson-databind:2.15.1")
+            library("jackson-datatype-jsr310", "com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.15.1")
+            library("jackson-kotlin", "com.fasterxml.jackson.module:jackson-module-kotlin:2.15.1")
 
             library("bcpkix-jdk15on", "org.bouncycastle:bcpkix-jdk15on:1.70")
             library("bcprov-jdk15on", "org.bouncycastle:bcprov-jdk15on:1.70")
 
-            library("cloudevents-api", "io.cloudevents:cloudevents-api:2.4.2")
-            library("cloudevents-json-jackson", "io.cloudevents:cloudevents-json-jackson:2.4.2")
-            library("cloudevents-core", "io.cloudevents:cloudevents-core:2.4.2")
+            library("cloudevents-api", "io.cloudevents:cloudevents-api:2.5.0")
+            library("cloudevents-json-jackson", "io.cloudevents:cloudevents-json-jackson:2.5.0")
+            library("cloudevents-core", "io.cloudevents:cloudevents-core:2.5.0")
             library("spring-cloud-function-context", "org.springframework.cloud:spring-cloud-function-context:4.0.2")
-            library("mailjet-client", "com.mailjet:mailjet-client:5.2.2")
+            library("mailjet-client", "com.mailjet:mailjet-client:5.2.3")
 
             library("pdfbox", "org.apache.pdfbox:pdfbox:2.0.28")
 
             library("mockito-kotlin", "org.mockito.kotlin:mockito-kotlin:4.1.0")
```

Comment:
5.0.0

Reply:
This new major release very fresh of the press - 14 hours ago apparently. [Github Readme still states 4.1](https://github.com/mockito/mockito-kotlin)

Nevertheless, I've changed it to 5.0.0

Reply:
ok